packages/suite/src/views/settings/SettingsGeneral/EnableViewOnly.tsx

```diff
@@ -33,9 +36,15 @@ export const EnableViewOnly = () => {
                 <ActionButton
                     onClick={handleSwitchDeviceClick}
                     variant="primary"
-                    isDisabled={!isDeviceConnected}
-                    isTooltipActive={!isDeviceConnected}
-                    tooltipContent={<Translation id="TR_SETTINGS_DEVICE_BANNER_TITLE_REMEMBERED" />}
+                    isDisabled={!isDeviceConnected || newlyConnectedDevice}
+                    isTooltipActive={!isDeviceConnected || newlyConnectedDevice}
+                    tooltipContent={
+                        newlyConnectedDevice ? (
```

Comment:
nitpick: You can use ternary only for setting value: 

```tsx
newlyConnectedDevice ? "TR_SETTINGS_DEVICE_VIEW_ONLY_DISABLED_TOOLTIP" : "TR_SETTINGS_DEVICE_BANNER_TITLE_REMEMBERED"

Reply:
what did we say to that `if-else?`
```ts
<if condition={newlyConnectedDevice}>
  <Then>
       <Translation id="TR_SETTINGS_DEVICE_VIEW_ONLY_DISABLED_TOOLTIP" />
  </Then>
  <Else>
      <Translation id="TR_SETTINGS_DEVICE_BANNER_TITLE_REMEMBERED" />
  </Else>
</If>
```

Reply:
Let's try it